src/main/java/com/novi/springboottechiteasy/dtos/televisiondtos/TelevisionInputDto.java

```diff
@@ -0,0 +1,209 @@
+package com.novi.springboottechiteasy.dtos.televisiondtos;
+
+import com.novi.springboottechiteasy.models.*;
+
+import java.util.Date;
+import java.util.List;
+
+public class TelevisionInputDto {
+    // id is geen input door generatedValue, dus die hoort er niet bij!
+
+    // validaties later verder aanvullen en testen.
+
+//    @NotNull(message = "Tv type is required")
+    private String type;
+//    @NotNull(message = "Tv brand is required")
+    private String brand;
+//    @Size(max = 20, message = "Tv names must stay between 0-20 characters")
+    private String name;
+//    @Positive(message = "Tv price can't be 0")
+    private Double price;
+    private List<ScreenSize> availableSizes;
+    private Double refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambiLight;
+//    @PositiveOrZero(message = "Initial stock can't have a negative value")
+    private Integer originalStock;
+    private Date originalStockDate;
+//    @PositiveOrZero(message = "Amount sold can't have a negative value")
+    private Integer sold;
+    private List<SoldDate> soldDates;
+    private Long remoteControllerId;
+    private Long compatibleModuleId;
+    private List<Long> compatibleWallBracketIds;
```

Comment:
Je hebt nu twee manieren om de relaties van Television met andere entiteiten te vullen. 

Je hebt in deze dto de velden waarin je de id's van andere entiteiten kunt meegeven. 
Je handeld dit ook netjes af in de service.

En je hebt de afzonderlijke PUT mappings als "assignRemoteControllerToTelevision()", die ook netjes worden afgehandeld in de service.

Je moet een manier kiezen, niet alles door elkaar gebruiken.